src/bioregistry/data/bioregistry.json

```diff
@@ -14087,6 +14087,34 @@
     },
     "uri_format": "https://www.cellbankaustralia.com/$1.html"
   },
+  "celldive": {
+    "cellosaurus": {
+      "category": "Cell line databases/resources",
+      "homepage": "https://celldive.dsmz.de/",
+      "name": "DSMZ Tools for Diving into Cell Line Data",
+      "prefix": "DSMZCellDive",
+      "uri_format": "https://celldive.dsmz.de/celllines/$1"
+    },
+    "contributor": {
+      "email": "cthoyt@gmail.com",
+      "github": "cthoyt",
+      "name": "Charles Tapley Hoyt",
+      "orcid": "0000-0003-4423-4370"
+    },
+    "description": "CellDive provides identifiers for authenticated human and animal cell lines, serving as reference models in biomedical research. These cell lines are characterized by various data types, including RNA-seq transcriptome profiles and short tandem repeat (STR) profiles, which ensure reproducibility and reliability. The resource enables researchers to identify and compare cell lines based on molecular and genomic features, supporting applications in cancer research, drug screening, and disease modeling",
+    "example": "ACC-1",
+    "github_request_issue": 1454,
+    "homepage": "https://celldive.dsmz.de/",
+    "mappings": {
+      "cellosaurus": "DSMZCellDive"
+    },
+    "name": "CellDive",
+    "pattern": "^ACC-\\d+$",
+    "synonyms": [
+      "DSMZCellDive"
```

Comment:
please maintain the old prefix dsmz as a synonym here

Reply:
In the latest iteration, `celldive` is curated as `part_of` `dsmz` while `dsmz` remains as its own entry so the synonym isn't needed, I think.